main.go

```diff
@@ -60,6 +60,9 @@ var (
 )
 
 func init() {
+	logs.InitLogs()
+	defer logs.FlushLogs()
```

Comment:
Is calling `defer` in the init function an intended change? This seems to changing the old behavior.

Reply:
Yeah, this might have been a miss step cause you are right, this means it's going to mask the problem. Basically `defer` and `os.Exit` don't work together so when we're exiting our binary it's actually not writing the logs, this moved it out of a func with `os.Exit` but I'm realizing that it's not going to actually defer anything now. Updating.

Reply:
Yeah, I'm just going to get rid of `klog` in the capn packages cause we don't even use it outside of `main.go` and we've been using controller-runtime's logger so it's duplicative.

Reply:
Actually we don't even use this log package, we're using klogr for the controller runtime logger so this isn't even used, deleting all of it :)